src/arrayMethodJoin.js

```diff
@@ -4,8 +4,20 @@
  * Implement method join
  */
 function applyCustomJoin() {
-  [].__proto__.join2 = function(separator) {
-    // write code here
+  [].__proto__.join2 = function(separator = ',') {
+    let strFromArr = '';
+
+    const sep = `${separator}`;
```

Comment:
do you need this?

Reply:
As I use slice, when I don't use interpolation I have an error 'TypeError: Cannot read property 'length' of null',  so I created an extra variable. But, As i recognised it can be too complivated, so I try to create a another way to solve this task